troubleshoot/monitoring/node-moves-outages.md

```diff
@@ -9,33 +9,93 @@ products:
+Potential causes of system maintenance include, but not limited to, situations like:
 
-::::{note}
-If you have only one master node in your cluster, during the master node vacate no notification will be sent. Kibana needs to communicate with the master node in order to send a notification. One way to avoid this is by shipping your deployment metrics to a dedicated monitoring cluster, which you can configure in Step 2 of [Getting notified about deployment health issues](../../deploy-manage/monitor/monitoring-data/configure-stack-monitoring-alerts.md) when you enable logging and monitoring.
+* A host where the Cloud Service Provider (CSP), like AWS, GCP, or Azure, has notified us in advance about hardware issues.
+* A host that has been abruptly terminated by the Cloud Service Provider (CSP) because of an underlying problem with their infrastructure.
+* Mandatory underlying host OS patch/upgrade activities for security/compliance reasons.
+* Other scheduled maintenance that can be found on the [Elastic status page](https://status.elastic.co/).
+
+Depending on the cause of the node movement, the behavior and expectations differ.
+
+* During planned operations, such as hardware upgrades or host patches, the system attempts to gracefully move the node to another host before shutting down the original one. This process allows shard relocation to complete ahead of time, minimizing any potential disruption.
+
+* In contrast, if a node’s host experiences an unexpected outage, the system automatically vacates the node and displays a related `Don't attempt to gracefully move shards` message on the [activity page](../../deploy-manage/deploy/elastic-cloud/keep-track-of-deployment-activity.md). Since the node is unavailable, the system skips checks that ensure the node’s shards have been moved before shutting down the node.
+
+### Will it cause an outage to my deployment?
+
+Unless overridden or unable, the system will automatically recover the vacated node’s data automatically from replicas or snapshots. If your cluster has [high availability (HA)](https://www.elastic.co/guide/en/cloud/current/ec-planning.html#ec-ha) configured, all search and indexing requests should continue to work within the reduced capacity until the node is replaced.
+
+Overall, having replicas and multiple availability zones helps minimize service interruption.
+
+### Under what circumstances could this result in data loss?
+
+The system maintenance process always attempts to recover the vacated node's data from replicas or snapshots. However, if the deployment is not configured with HA, including replica shards, the maintenance process may not be able to recover the data from the vacated node.
+
+To minimize this risk, ensure your deployment follows the [high availability best practices](/deploy-manage/deploy/elastic-cloud/elastic-cloud-hosted-planning.md#ec-ha), which recommend:
```

Comment:
same as https://github.com/elastic/docs-content/pull/1619/files#r2129040209, the anchor does not exist anymore

Reply:
Yes, this one is I added the anchor so it's good. Thanks for checking!